### ISSUE TITLE
Support for winrm connection to Windows hosts

### ISSUE THREAD

**Issue by jhampson-dbre** [ansible/ansible-runner-service#75]
We were looking at creating an API wrapper around ansible runner when we found this project. 🤗

Our use case primarily revolves around managing Windows servers using winrm rather than SSH. Does ansible runner service inventory support Windows hosts that use winrm connection?

**Comment by mnecas** [comment]
Hi,
don't know much about winrm but...
If I understand it correctly you need to have config like this [1] in extravars as in [2]?
You can send to the body request JSON data which will be used in the extravars.
You can check the extravars in _private_data_dir_/artifacts/_UUID_/env/

[1] https://docs.ansible.com/ansible/latest/user_guide/windows_winrm.html#basic
[2] https://github.com/ansible/ansible-runner/blob/master/demo/env/extravars

**Comment by jhampson-dbre** [comment]
Usually we have it configured as group_vars in inventory, such as
```
# group_vars/windows.yml
ansible_connection: winrm
ansible_winrm_transport: kerberos
```
It seems like we should be able to do the same here and set disable ssh_checks in config.yaml, since no SSH keys will be involved?
```
ssh_checks: False
```

Would you take a pull request if any documentation is needed for windows-specific setup?

**Comment by mnecas** [comment]
Yeah, it would be best to have it in docs (not sure where specifically) don't hesitate to create PR.

**Comment by gthieleb** [comment]
@jhampson-dbre Do you already have some decision where to put configuration hints for winrm communication? I can't find something the documentation.
